src/app/wizard/step/provider-settings/provider/extended/azure/template.html

```diff
@@ -22,6 +22,15 @@
       Enter your credentials to load autocompletions.
     </ng-container>
   </km-autocomplete>
+  <km-autocomplete label="VNet Resource Group"
+                   [formControlName]="Controls.VNetResourceGroup"
+                   [isLoading]="isLoadingResourceGroups"
+                   [options]="resourceGroups">
```

Comment:
Just as a question, because I'm not into this topic:
Is it wanted, that `VNet Resource Group` has the same options as `Resource Group`?

Reply:
Yes, I have confirmed that. It should work like this.